client/recipe/core/impl/src/commonMain/kotlin/com/plusmobileapps/chefmate/recipe/core/impl/detail/RecipeDetailBlocImpl.kt

```diff
@@ -49,6 +54,20 @@ class RecipeDetailBlocImpl(
                 isDeleting = it.isDeleting,
                 showDeleteConfirmationDialog = it.showDeleteConfirmationDialog,
                 recipe = it.recipe,
+                createdAt =
+                    FixedString(
+                        dateTimeFormatter.formatDateTime(
+                            instant = it.recipe.createdAt,
+                            timeZone = dateTimeUtil.currentTimezone,
+                        ),
+                    ),
+                updatedAt =
+                    FixedString(
+                        dateTimeFormatter.formatDateTime(
+                            instant = it.recipe.updatedAt,
+                            timeZone = dateTimeUtil.currentTimezone,
+                        ),
+                    ),
             )
```

Comment:
- [ ] look at adding these methods in the date time util directly to avoid needing to inject two dependencies to get the time zone